### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by timo-piirmann**
* Don't log error if taxon name not found
* Show less intimidating page in case of 404
* Code cleanups

### CODE REVIEW

File: grails-app/views/404.gsp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
+    <head>
+    </head>
+
+    <body>
+        <center>
```
Comment (raivop) [inline grails-app/views/404.gsp:6]: The center tag is obsolete in HTML5.

File: grails-app/views/404.gsp [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            <h1>404</h1>
+
+            <g:if test="${message}">
+                ${message}
+            </g:if>
+            <g:else>
+                Well, stuff can go wrong sometimes and maybe you mistyped a URL, tapped/clicked on a broken link or your
+                computer just wants to drive you nuts. Anyway we are really sorry … keep calm and try again.
```
Comment (raivop) [inline grails-app/views/404.gsp:14]: There should be a comma after "Anyway".

Comment (raivop) [reply grails-app/views/404.gsp:14]: Not even sure what's up with …

Comment (timo-piirmann) [reply grails-app/views/404.gsp:14]: Create a valid 404 or go away

**Pull request merged**